doc/api/process.markdown

```diff
@@ -169,15 +185,18 @@ event tells you when the list of unhandled rejections shrinks.
 For example using the rejection detection hooks in order to keep a list of all
 the rejected promises at a given time:
 
-    var unhandledRejections = [];
+    var unhandledRejections = new Set();
     process.on('unhandledRejection', function(reason, p) {
-        unhandledRejections.push(p);
+      unhandledRejections.add(p);
     });
     process.on('rejectionHandled', function(p) {
-        var index = unhandledRejections.indexOf(p);
-        unhandledRejections.splice(index, 1);
+      unhandledRejections.delete(p);
     });
 
+You could then record this list in some error log, either periodically or upon
+process exit. This would avoid the false positive seen in the above resource
```

Comment:
It's misleading but it's not a false positive. Since the process has terminated nothing else can handle the rejection so it's just a positive. It's still problematic in that it's not useful in this case.